tools/cross_runner/src/layers_utils.h

```diff
@@ -23,11 +23,12 @@ struct TensorShape
     std::uint32_t d = 0; // for 5d tensors
     std::uint32_t h = 0;
     std::uint32_t w = 0;
+    bool nhwc = 0;
```

Comment:
Tensor shape should be layout agnostic
It's just shape (dimensions)
It shouldnt know about layout.